app/controllers/teams_controller.rb

```diff
@@ -14,6 +14,36 @@ def switch
     redirect_to root_path
   end
 
+  def manage_users
+    team = current_user.teams.friendly.find params[:team_id]
+    authorize team
+    @users = team.users.order(:name).map do |user|
+      Admin::UserPresenter.new(user)
+    end
+  end
+
+  def find_user_by_email
+    authorize current_team
```

Comment:
This action only redirect view where the user will insert the email. I wanted leave this empty method but the Policy not permited.